src/components/DropMenu/examples/3.disabled.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import DropMenuStateless from '../DropMenu';
+import { buildStatefulComponent } from '../../../util/state-management';
+import Button from '../../Button/Button';
+
+const DropMenu = buildStatefulComponent(DropMenuStateless);
+const {
+	Control,
+	Option
+} = DropMenu;
+
+export default React.createClass({
+	render() {
+		return (
+			<DropMenu isDisabled>
```

Comment:
I was curious about that: it was a bit confusing to see on the examples page that there's a disabled example where the button is not disabled.